packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html

```diff
@@ -0,0 +1,105 @@
+<div class="p-5">
+    <h3>
+        <strong
+            data-translate="telephony_line_softphone_add_my_device"
+        ></strong>
+    </h3>
+    <p>
+        <b
+            data-translate="telephony_line_softphone_add_device_description_bold"
+        ></b>
+        <span
+            data-translate="telephony_line_softphone_add_device_description"
+        ></span>
+    </p>
+    <div
+        class="d-flex flex-wrap w-100 minw-0 align-items-center justify-content-between"
+    >
+        <form class="form-inline" name="$ctrl.softphoneAddDeviceForm">
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    data-help-text="{{:: 'telephony_line_softphone_generate_warning' | translate }}"
+                    label="{{ ::'telephony_line_softphone_add_device_label' | translate }}"
+                    size="sm"
+                >
+                    <div class="input-group">
+                        <input
+                            type="text"
+                            id="input_name"
+                            name="input_name"
+                            class="oui-input"
+                            data-ng-model="$ctrl.input.name"
+                            data-ng-pattern="/^[a-zA-Z0-9\-\_]+$/"
```

Comment:
```suggestion
                            data-ng-pattern="$ctrl.FIELD_NAME_REGEX"
```
you could move this regex in constant file